src/main.rs

```diff
@@ -2018,7 +2018,7 @@ mod tests {
         }
 
         fn api_create_body(&self, cpu_count: u8) -> String {
-            format! {"{{\"cpus\":{{\"boot_vcpus\":{},\"max_vcpus\":{}}},\"kernel\":{{\"path\":\"{}\"}},\"cmdline\":{{\"args\": \"\"}},\"net\":[{{\"ip\":\"{}\", \"mask\":\"255.255.255.0\", \"mac\":\"{}\"}}], \"disks\":[{{\"path\":\"{}\"}}, {{\"path\":\"{}\"}}]}}",
+            format! {"{{\"cpus\":{{\"boot_vcpus\":{},\"max_vcpus\":{}}},\"kernel\":{{\"path\":\"{}\"}},\"cmdline\":{{\"args\": \"\"}},\"net\":[{{\"ip\":\"{}\", \"mask\":\"255.255.255.0\", \"mac\":\"{}\", \"num_queues\":2, \"queue_size\":256}}], \"disks\":[{{\"path\":\"{}\"}}, {{\"path\":\"{}\"}}]}}",
```

Comment:
Sorry to insist but I don't understand why you need to add this. Because you defined some default values for both `num_queues` and `queue_size` for serde, things should simply work without updating this.

Reply:
Re-run the test case without adding the two options, CI has passed, so I think we can safely remove them. I'm not sure why it did not work previously when not define them, but glad to see it works now.